core/psa_crypto.c

```diff
@@ -8817,7 +8817,7 @@ psa_status_t psa_pake_setup(
     }
 
     if (PSA_ALG_IS_PAKE(cipher_suite->algorithm) == 0 ||
```

Comment:
Note on the commit message: "rebase" does not make sense for a commit description. Please use a descriptive commit message, for example "Move the PAKE hash into the alg ID".

(Note for future PRs: this commit is a bit too big. In retrospect it would have been better to start with a preliminary commit that introduces `PSA_ALG_IS_JPAKE` and uses it everywhere, and only then make the actual change. Well, it's always easier in hindsight. Also, I'm not asking you to change the commit structure here (just the message), so that's just advice for the future.)